class test2
{
	public static void main(String args[])
	{
		try 
		{

		}
		catch(ArithmeticException e)
		{

		}
		catch(Exception e)
		{
			
		}
	}
}